modules/vault-cluster/variables.tf

```diff
@@ -80,6 +80,12 @@ variable "cluster_tag_key" {
   default     = "Name"
 }
 
+variable "cluster_extra_tags" {
+  description = "A list of additional tags to add to each Instance in the ASG."
+  type = "list"
+  default = []
```

Comment:
Could you update the `description` to mention that each element in the list should be a map with the keys `key`, `value`, and `propagate_at_launch`? Also, an example in comments above `default` would be helpful.